extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerRemoveCoverageCommand.java

```diff
@@ -52,12 +56,17 @@
 	@Override
 	public boolean prepare(
 			OperationParams params ) {
-		super.prepare(params);
```

Comment:
`super.prepare(params);` got eliminated, it seems like it needs to be there (at least for the CLI to work the same, presumably the services should be similar too).